src/diem/testing/miniwallet/app/app.py

```diff
@@ -16,14 +16,24 @@
 
 
 class Base:
-    def __init__(self, account: LocalAccount, client: jsonrpc.Client, name: str, logger: logging.Logger) -> None:
+    def __init__(
+        self,
+        account: LocalAccount,
+        child_accounts: List[LocalAccount],
+        client: jsonrpc.Client,
+        name: str,
+        logger: logging.Logger,
+    ) -> None:
         self.logger = logger
-        self.diem_account = DiemAccount(account, client)
+        self.diem_account = DiemAccount(account, child_accounts, client)
```

Comment:
are child_accounts potentially optional? If so should use named arguments (easier to change signature without rewriting all invocations)

Reply:
It is not optional, but you can use empty array to make it like optional, thus it is not necessary to make it optional.
DiemAccount is used for managing Diem accounts for mini-wallet app, it was not designed for sharing or reuse, thus current simple constructor is better.

Reply:
I would argue if you want people to pass in `[]`, then it should be a named optional param, but it's a nit :-)